Makefile

```diff
@@ -20,14 +17,23 @@ ifdef package
 else
 	$(GINKGO) --randomizeAllSpecs -p --skipPackage=tests ./...
 endif
-		
 
-build: enterprise-prepare-build ## Build rudder-server binary
-	$(GO) build -o bin/rudder-server main.go
+build-sql-migrations: ./services/sql-migrator/migrations_vfsdata.go ## Prepare sql migrations embedded scripts	
+
+prepare-build: build-sql-migrations enterprise-prepare-build
```

Comment:
We need to add non file targets like prepare-build, build-sql-migrations in .PHONY

Reply:
That's true, I'll add them